### ISSUE TITLE
PATCH vs PUT

### ISSUE THREAD

**Issue by hedder** [jakubrohleder/angular-jsonapi#22]
Of.spec claims to send PATCH request to update existing records instead of PUT.
It also says that only updated attributes/relationships should be sent.

> Updating a Resource's Attributes
> PATCH /articles/1 HTTP/1.1

<a href="http://jsonapi.org/format/#crud-updating-resource-attributes">link</a>

Also, it could significantly decrease amount of data we're sending to the back-end.

**Comment by ghost** [comment]
+1, PUT is not part of the jsonapi.org spec: http://jsonapi.org/faq/#wheres-put

**Comment by mkalpana** [comment]
+1. PATCH should be used for updating existing records instead of PUT

**Comment by yves-s** [comment]
Yes, it would be nice to see a patch method. We just hacked a patch methode in the build file to fulfill the jsonapi.org requirements and update relations.
